### DISCUSSION TITLE
game bot guess my number

### DISCUSSION THREAD

**Discussion by Vladke2** [cpp-for-yourself/lectures-and-homeworks#63] [General]
please help create a code for the game bot guess my number

**Comment by niosus** [comment]
It would help to know what you tried already and what doesn't work. Remember that the process of writing the homework code is important, not the end result. You _are_ learning afterall.

**Comment by Vladke2** [comment]
I managed to get the user's number, but I can't get the bot to guess it

**Reply by niosus** [reply]
I am not sure we are talking about the same thing. What do you mean by:
> I managed to get the user's number

Where did you get the number from?

**Reply by Vladke2** [reply]
Thanks for trying to help, but I got what I wanted.

**Comment by niosus** [comment]
Closing as not an issue anymore.
